core/src/State/FocusedElement.ts

```diff
@@ -410,7 +410,17 @@ export class FocusedElementState
                 // Nowhere to move inside the current Mover.
                 e.preventDefault(); // We don't need the page to scroll when we're custom-handling
                                     // the arrows.
-                return;
+
+                if (!ctx.moverOptions?.cyclic) {
+                    return;
+                }
+
+                // cyclic navigation, focus first or last elements in the mover container respectively
```

Comment:
This is assuming that first/last focusable are first/last direct descendants of the container. Which mostly won't be the case in the real scenarios.

There is an example how to find the first/last focusable inside the container above:
```? (isPrev // Find next focusable outside of the Mover container.
                    ? this._ah.focusable.findFirst(ctx.mover)
                    : this._ah.focusable.findLast(ctx.mover)```

Reply:
Ah i didn't know there was an API in focusable for this, will modify